pytds/tds.py

```diff
@@ -3180,6 +3180,8 @@ def make_param(self, name, value):
         if value is default:
             column.flags |= fDefaultValue
             value = None
+            if value_type is _Default:
```

Comment:
This check is necessary because value_type can be overriden for output
parameters.

On Thu, Feb 5, 2015 at 6:35 PM, denisenkom <notifications@github.com> wrote:

> In pytds/tds.py
> <https://github.com/denisenkom/pytds/pull/45#discussion_r24210716>:
>
> > @@ -3180,6 +3180,8 @@ def make_param(self, name, value):
> >          if value is default:
> >              column.flags |= fDefaultValue
> >              value = None
> > +            if value_type is _Default:
>
> this seems like redundant test, value is already an instance of _Default
> hence value_type is _Default, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/denisenkom/pytds/pull/45/files#r24210716>.
>



-- 
Regards,
  Valerii Lashmanov